modules/understanding-node-cves-scanner-v4.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities/common-vuln-management-tasks.adoc
+
+:_mod-docs-content-type: CONCEPT
+// DO NOT RENAME THE ID - there are UI links using it in ACS
+[id="understanding-node-cves-scanner-v4_{context}"]
+= Understanding differences in scanning results between the Stackrox Scanner and Scanner V4
+
+== Summary
+
+Scanning {op-system} node hosts with Scanner V4 reports significantly more CVEs (~390) compared to StackRox Scanner (~50) for the same operating system version. A manual review of selected vulnerabilities revealed that:
+
+* The VEX data used in Scanner V4 is more accurate.
+* The VEX data includes granular statuses, such as "no fix planned" and "fix deferred".
+* Some vulnerabilities reported by StackRox Scanner were false positives.
+
+As a result, Scanner V4 provides a more accurate and realistic vulnerability assessment.
+
+Users may find discrepancies in reported vulnerabilities surprising—especially if some secured clusters still use older {product-title-short} versions with StackRox Scanner while others have transitioned to Scanner V4. This article explains these differences and provides guidance on how to manually verify reported vulnerabilities if needed.
+
+== Discrepancies in reported vulnerabilities
+
+We analyzed the differences in reported CVEs for three recent, arbitrarily selected {op-system} versions. This article presents findings for {op-system} version `417.94.202501071621-0`.
+For this version:
+
+* StackRox Scanner reports 49 CVEs.
+* Scanner V4 reports 389 CVEs.
+
+The breakdown is as follows:
+
+* 1 CVE is reported only by the StackRox Scanner.
+* 48 CVEs are reported by both scanners.
+* 341 CVEs are reported only by Scanner V4.
+
+=== CVEs reported only by the StackRox Scanner
+
+The single CVE reported exclusively by StackRox Scanner was a false positive. CVE-2022-4122 was flagged for the `podman` package in version `5:5.2.2-1.rhaos4.17.el9.x86_64`. However, a manual review of VEX data from link:https://access.redhat.com/errata/RHSA-2024:9102[RHSA-2024:9102] indicated that this vulnerability was fixed in version `5:5.2.2-1.el9`. Therefore, the package version scanned was the first to contain the fix and was no longer affected.
+
+=== CVEs reported only by Scanner V4
+
+We randomly selected 10 CVEs from the 341 unique to Scanner V4 and conducted a detailed analysis using VEX data. The vulnerabilities fell into two categories:
+
+. Affected packages with a fine-grained status indicating that no fix is planned.
+. Affected packages with no additional status details regarding a fix.
```

Comment:
```suggestion
* Affected packages with no additional status details regarding a fix
```
These don't need to be numbered because the [sequence isn't significant](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#unordered-lists) and they are not complete sentences so do not need punctuation - see [punctuation](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#punctuation-in-lists)